drivers/display/display_gc9a01a.c

```diff
@@ -0,0 +1,599 @@
+/**
+ * Copyright (c) 2017 Jan Van Winkel <jan.van_winkel@dxplore.eu>
```

Comment:
Don't copy other peoples copyright. Happened today to me too 
:^)

Reply:
Thanks for the heads up. will remove with the next commit.